### PR TITLE
Change icons for cover with device_class curtain

### LINKED ISSUE #11706: Curtain commands shown as cover
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I have installed Switchbot curtains, but the controls are shown as a cover, but they are curtains.
So you expect to open the curtains sidewise and no up and down.

### Describe the behavior you expected

Current situation:
image

Maybe it's an idea to use mdi:curtains and mdi:curtains-closed for the curtain controls?
image

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

core-2022.2.7

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

_No response_

### Which operating system are you using to run this browser?

_No response_

### State of relevant entities

```txt
The entity has the following state class:
device_class: curtain
```

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pascalwinters**
## Proposed change
Before:
image

After:
image

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #11706
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by pascalwinters** [comment]
> I'm not sure if this makes it any clearer. I think the horizontal arrows like used for gate are better suited.

I understand your remark, but this one is more curtain specific. So it feels more accurate. And it's correlates with the status icon.
The only 'problem', it doesn't look like an action.

**Comment by bramkragten** [comment]
> The only 'problem', it doesn't look like an action.

Yeah, and that's a pretty big problem I would say

**Comment by pascalwinters** [comment]
> horizontal arrows like used for gate are better suited.

Ok, shall I apply the horizontal arrows like used for gate?

File: src/common/entity/cover_icon.ts [hunk 1]
```diff
@@ -121,6 +121,8 @@ export const computeOpenIcon = (stateObj: HassEntity): string => {
     case "door":
     case "gate":
       return mdiArrowExpandHorizontal;
+    case "curtain":
+      return mdiArrowExpandHorizontal;
```
Comment (bramkragten) [inline src/common/entity/cover_icon.ts:125]:
```suggestion
    case "door":
    case "gate":
    case "curtain":
      return mdiArrowExpandHorizontal;
```

File: src/common/entity/cover_icon.ts [hunk 2]
```diff
@@ -132,6 +134,8 @@ export const computeCloseIcon = (stateObj: HassEntity): string => {
     case "door":
     case "gate":
       return mdiArrowCollapseHorizontal;
+    case "curtain":
+      return mdiArrowCollapseHorizontal;
```
Comment (bramkragten) [inline src/common/entity/cover_icon.ts:138]:
```suggestion
    case "door":
    case "gate":
    case "curtain":
      return mdiArrowCollapseHorizontal;
```

Comment (pascalwinters) [reply src/common/entity/cover_icon.ts]: Good point. Resolved.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**